sdk/tables/swagger.md

```diff
@@ -0,0 +1,41 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+<!-- Original autorest command used by Chris Scott -->
+<!-- autorest --use=@autorest/go@4.0.0-preview.20 https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cosmos-db/data-plane/readme.md --tag=package-2019-02 --file-prefix="zz_generated_" --modelerfour.lenient-model-deduplication --license-header=MICROSOFT_MIT_NO_VERSION --output-folder=aztables --module=aztables --openapi-type="data-plane" --credential-scope=none -->
+
+``` yaml
+go: true
+version: "^3.0.0"
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/d744b6bcb95ab4034832ded556dbbe58f4287c5b/specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: false
```

Comment:
Half of this belongs in a common swagger that gets required, but we can extract that out later when there are two copies.